crypto/nrf_cc310_platform/include/nrf_cc3xx_platform_kmu.h

```diff
@@ -144,5 +219,53 @@ int nrf_cc3xx_platform_kdr_load_key(uint8_t key[16]);
 
 #endif // defined(NRF52840_XXAA)
 
+/** @brief Function to use CMAC to derive a key stored in KMU/KDR
+ *
+ * @details The KDF is using a PRF function described in the Special publication
+ *          800-108: Recommendation for Key Derivation Using Pseudorandom Functions
+ *          https://csrc.nist.gov/publications/detail/sp/800-108/final.
+ *
+ *          This algorithm is described in chapter 5.1 - KDF in Counter Mode
+ *
+ *          The format of the PRF (the input) is as follows:
+ *          PRF (KI, i || Label || 0x00 || Context || L)
+ *
+ *          KI: The Key derivation key
+ *          i : The counter value for each iteration of the PRF represented
+ *              as one byte.
+ *          label: A string identifying the purpose of the derived key
+ *                 that is up to 64 bytes long.
+ *          0x00: A single byte delimiter.
+ *          Context: Fixed information about the derived keying material
+ *                   that is up to 64 bytes long.
+ *          L : The length of derived key material in bits represented as two
+ *              bytes.
```

Comment:
```suggestion
 *          L : The length of derived key material in bits represented in big-endian
 *              format with as many bytes as is needed to represent the number.
```

Reply:
This will be fixed in 0.9.10 version of nrf_cc3xx_platform